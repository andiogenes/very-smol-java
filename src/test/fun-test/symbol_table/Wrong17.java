class Main {
    void main() {
        int x = 5;
        switch (x) {
            x = 10;
        }
    }
}