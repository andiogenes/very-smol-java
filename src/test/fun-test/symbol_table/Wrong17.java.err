Parsing error: switch case expected, got: 'x' at line 5, pos 13
