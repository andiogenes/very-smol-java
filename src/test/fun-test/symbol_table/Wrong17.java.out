digraph G {
	n0 [label="Class(Main)"];
	n0 -> n1;
	n0 -> n2;
	n1 [label="left", style=filled, fillcolor=gray];
	n2 [label="Method(main,VOID)"];
	n2 -> n5;
	n2 -> n6;
	n5 [label="left", style=filled, fillcolor=gray];
	n6 [label="Variable(x,INT,5)"];
	n6 -> n13;
	n6 -> n14;
	n13 [style=filled, fillcolor=black];
	n13 -> n27;
	n13 -> n28;
	n27 [label="left", style=filled, fillcolor=gray];
	n28 [label="right", style=filled, fillcolor=gray];
	n14 [label="right", style=filled, fillcolor=gray];
}
